roles/postgres/tasks/main.yml

```diff
@@ -62,20 +62,35 @@
     ansible_ssh_pipelining: yes
   block:
     - name: Create necessary databases
-      postgresql_db:
+      community.postgresql.postgresql_db:
         name: "{{ item }}"
       loop:
         - "{{ repo_db_name }}"
         - "{{ sync_db_name }}"
 
-    - name: Set users privileges on databases
-      postgresql_user:
+    # https://dba.stackexchange.com/a/35317/5179
+    - name: Revoke public access to databases
+      community.postgresql.postgresql_privs:
+        db: "{{ item }}"
+        privs: ALL
```

Comment:
This (and the fact we restart the services during play) enforces someone have to run the playbook during maintenance schedules only. That's an acceptable constraint but we should document it now maybe.